ui/desktop/tests/integration/components/settings-card/user-test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+import Service from '@ember/service';
+
+const data = {
+  authenticated: {
+    username: 'admin',
+    authenticator: 'authenticator:password',
+  },
+};
+
+module('Integration | Component | settings-card/user', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  test('it renders password correctly', async function (assert) {
+    this.owner.register(
+      'service:session',
+      class extends Service {
+        data = data;
+        isAuthenticated = true;
+      },
+    );
+
+    await render(hbs`<SettingsCard::User />`);
+
+    assert.dom('.hds-badge__text').hasText('Authenticated');
+    assert.dom('[data-test-auth-method]').hasText('Password');
+    assert.dom('[data-test-username]').hasText('admin');
```

Comment:
non-blocking: Could we move these to const variables in this file? This will help during test selector refactor work in the future.

Reply:
Sure, I tried not to extract to variables since I expected them only to be used once but I can still move them to consts